### PR TITLE
dx-533/add-warning-to-property-type

### PR DESCRIPTION
**Pull request by Isengo1989**

### CODE REVIEW

**Comment by AydinHassan** [comment]
Maybe it's easier to just say the properties should not be private ?

File: guides/plugins/plugins/framework/data-handling/add-custom-complex-data.md [hunk 1]
```diff
@@ -191,6 +191,10 @@ Yet, we highly recommend you to create a custom `Entity` class, as well as a cus
 
 The entity class itself is a simple key-value object, like a struct, which contains as many properties as fields in the definition, ignoring the ID field, which is handled by the `EntityIdTrait`.
 
+::: warning
+The properties of your entity class have to be at least `protected`, otherwise the data abstraction layer won't be able to set the values.
```
Comment (Isengo1989) [inline guides/plugins/plugins/framework/data-handling/add-custom-complex-data.md:195]:
```suggestion
Properties of your entity class shouldn't be `private` so the data abstraction layer will be able to set the values.
```

Comment (Isengo1989) [reply guides/plugins/plugins/framework/data-handling/add-custom-complex-data.md:195]: @AydinHassan yup, sounds better and is more clear. Good to go?

Comment (AydinHassan) [reply guides/plugins/plugins/framework/data-handling/add-custom-complex-data.md:195]: yep nice

**Review by AydinHassan — APPROVED** [review]

**Pull request merged**